roles/result/tasks/fio-samples.yml

```diff
@@ -0,0 +1,96 @@
+---
+
+- name: Check that the log files exist
+  find:
+    paths: "{{item.path}}"
+    patterns: "fio"
+    use_regex: True
+  register: fio_files_found
+
+- block:
+  - set_fact:
+      fio_output_current_sample: "{{ lookup('file', item.path + '/' + 'fio-result.json') }}"
+
+  - set_fact:
+      fio_start_time: "{{ fio_output_current_sample.timestamp_ms }}"
+
+  - debug:
+      msg: "Start time for current fio sample was {{ fio_start_time }} for uuid {{ uuid }}"
+
+  - name: Index fio result data
+    uri:
+      url: "http://{{ elasticsearch.server }}:{{ elasticsearch.port }}/{{ elasticsearch.index }}-result/doc"
+      method: POST
+      status_code: 201
+      body: "{{ lookup('template','fio.json.j2') }}"
+      body_format: json
+
+  # read_csv module was introduced in ansible version 2.8, and so can't be used at the time of
```

Comment:
considering this is all commented out waiting for 2.8, can we just remove it, until the operator ships with 2.8?

Reply:
Operator should ship with ansible 2.8 soon, as the rc2 is already out for ansible 2.8. Had the conversation with ansible-operator folks, and they mentioned they're busy with 4.1 release and thats why couldn't find time. I'd say let's leave it in there, as i'll soon open a PR once operator ships with 2.8. I'm just concerned that, 2.8 will be out and we'll lose track of it somehow.